README.md

```diff
@@ -32,47 +33,70 @@ Build can be done by running:
 
     pip install -e .
 
+The DiffKemp binary is then located in `bin/diffkemp`.
```

Comment:
This might be a good place to mention how DiffKemp works in respect to current working directory and `SIMPLL_BUILD_DIR`.

Reply:
We may rethink this soon, so I'd keep it as it is for now.